outbound-dialer/src/main/java/org/motechproject/nms/outbounddialer/web/OutboundDialerController.java

```diff
@@ -0,0 +1,147 @@
+package org.motechproject.nms.outbounddialer.web;
+
+import org.motechproject.nms.outbounddialer.web.contract.FileProcessedStatusRequest;
+import org.motechproject.nms.outbounddialer.web.contract.CdrFileNotificationRequest;
+import org.motechproject.nms.outbounddialer.web.contract.CdrFileNotificationRequestFileInfo;
+import org.motechproject.nms.outbounddialer.web.contract.BadRequest;
+import org.motechproject.nms.outbounddialer.service.OutboundDialerService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.converter.HttpMessageNotReadableException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import java.util.regex.Pattern;
+
+/**
+ * OutboundDialerController
+ */
+@Controller
+public class OutboundDialerController {
+
+    public static final String NOT_PRESENT = "<%s: Not Present>";
+    public static final String INVALID = "<%s: Invalid>";
+    public static final Pattern TARGET_FILENAME_PATTERN = Pattern.compile("OBD_NMS[1-9]_20[0-9]{12}\\.csv");
```

Comment:
We (MOTECH) generate target files and 'send' them to the IVR system.
We receive CDR files (summary & detailed) from the IVR syatsm. So this should be either CDR_SUMMARY_FILENAME_PATTERN or CDR_DETAILED_FILENAME_PATTERN and check conformity with point 14 of section 4.1.5 in the spec. (ie: CDR_Summary_xxx or CDR_Detail_xxx)